notebook/i18n/nl/LC_MESSAGES/notebook.po

```diff
@@ -0,0 +1,607 @@
+msgstr "Het bestand waarin het cookiegeheim wordt opgeslagen."
+
+#: notebook/notebookapp.py:628
+#, python-format
+msgid "Writing notebook server cookie secret to %s"
+msgstr "Cookiegeheim voor notebookserver schrijven naar %s"
+
+#: notebook/notebookapp.py:635
+#, python-format
+msgid "Could not set permissions on %s"
+msgstr "Kan geen machtigingen instellen op %s"
+
+#: notebook/notebookapp.py:640
+msgid ""
+"Token used for authenticating first-time connections to the server.\n"
+"\n"
+"        When no password is enabled,\n"
+"        the default is to generate a new, random token.\n"
+"\n"
+"        Setting to an empty string disables authentication altogether, which is NOT RECOMMENDED.\n"
+"        "
+msgstr ""
+"Token wordt gebruikt voor het verifiëren van eerste verbindingen met de server.\n"
+"\n"
+"Wanneer er geen wachtwoord is ingeschakeld,\n"
+"        de standaardinstelling is het genereren van een nieuwe, willekeurige token.\n"
+"\n"
+"Als u een lege tekenreeks instelt, wordt de verificatie helemaal uitgeschakeld, wat niet wordt aanbevolen.\n"
+"        "
+
+#: notebook/notebookapp.py:650
+msgid ""
+"One-time token used for opening a browser.\n"
+"        Once used, this token cannot be used again.\n"
+"        "
+msgstr ""
+"Eenmalige token die wordt gebruikt voor het openen van een browser.\n"
+"        Eenmaal gebruikt, kan dit token niet opnieuw worden gebruikt.\n"
+"        "
+
+#: notebook/notebookapp.py:726
+msgid ""
+"Specify Where to open the notebook on startup. This is the\n"
+"        `new` argument passed to the standard library method `webbrowser.open`.\n"
+"        The behaviour is not guaranteed, but depends on browser support. Valid\n"
+"        values are:\n"
+"            2 opens a new tab,\n"
+"            1 opens a new window,\n"
+"            0 opens in an existing window.\n"
+"        See the `webbrowser.open` documentation for details.\n"
+"        "
+msgstr ""
+"Geef op waar u het notebook moet openen bij het opstarten. Dit is de\n"
+"        'nieuw' argument doorgegeven aan de standaard bibliotheek methode 'webbrowser.open'.\n"
+"        Het gedrag is niet gegarandeerd, maar is afhankelijk van browserondersteuning. Geldig\n"
+"        waarden zijn:\n"
+"            2 opent een nieuw tabblad,\n"
+"            1 opent een nieuw venster,\n"
+"            0 wordt geopend in een bestaand venster.\n"
+"        Zie de documentatie 'webbrowser.open' voor meer informatie.\n"
+"        "
+
+#: notebook/notebookapp.py:737
+msgid "DEPRECATED, use tornado_settings"
+msgstr "DEPRECATED, gebruik tornado_settings"
+
+#: notebook/notebookapp.py:742
+msgid ""
+"\n"
+"    webapp_settings is deprecated, use tornado_settings.\n"
+msgstr ""
+"\n"
+"webapp_settings is deprecated, gebruik tornado_settings.\n"
+
+#: notebook/notebookapp.py:746
+msgid ""
+"Supply overrides for the tornado.web.Application that the Jupyter notebook "
+"uses."
+msgstr ""
+"Levering overschrijft voor de tornado.web.Application die de Jupyter "
```

Comment:
Weird sentence

Reply:
"Supply overrides voor de tornado.web.Application die de Jupyter"
" notebook gebruikt." is this better?
I'm not sure what the sentence is referring too in this case. The line numbering seems to be out of sync.

Reply:
I have no idea as well.

If I search for it in the code I get:
https://github.com/jupyter/notebook/blob/b4a2ad990a49479965b681ddc89ab0a9e993a2fc/notebook/notebookapp.py#L1001

From cotext it seems like a user can change the default settings for the tornado.web.Application by supplying some extra config. So maybe a translation like:
"Geef extra instellingen voor de 'tornado.web.Application' die gebruikt wordt door de Jupyter notebook."
could work.

Reply:
agreed.